pkgs/development/ocaml-modules/biniou/default.nix

```
@@ -1,29 +1,21 @@
-{ lib, fetchFromGitHub, buildDunePackage, easy-format }:
+{ lib, fetchurl, buildDunePackage, camlp-streams, easy-format }:
 
 buildDunePackage rec {
   pname = "biniou";
-  version = "1.2.1";
+  version = "1.2.2";
 
-  useDune2 = true;
-
-  src = fetchFromGitHub {
-    owner = "ocaml-community";
-    repo = pname;
-    rev = version;
-    sha256 = "0x2kiy809n1j0yf32l7hj102y628jp5jdrkbi3z7ld8jq04h1790";
+  src = fetchurl {
+    url = "https://github.com/mjambon/biniou/releases/download/${version}/biniou-${version}.tbz";
```

```suggestion
    url = "https://github.com/ocaml-community/biniou/releases/download/${version}/biniou-${version}.tbz"
```

https://github.com/ocaml-community/biniou/releases/download/1.2.2/biniou-1.2.2.tbz is available
Also general question on nixpkgs do we prefer GitHub tag or release ?